test/runtest.py

```diff
@@ -1330,8 +1330,10 @@ def testLoadBalancing(self):
     # in a roughly equal manner.
     num_workers = 21
```

Comment:
Why don't we also change `num_cpus` to `7` to make it consistent with what the test was doing before.